crates/red_knot_python_semantic/tests/mdtest.rs

```diff
@@ -18,12 +19,19 @@ fn mdtest(fixture: Fixture<&str>) {
 
     let test_name = test_name("mdtest", absolute_fixture_path);
 
+    let output_format = if std::env::var("GITHUB_ANNOTATIONS_FORMAT").is_ok() {
```

Comment:
Nit: Can we use something like `MDTEST_OUTPUT_FORMAT` or `MDTEST_GITHUB_OUTPUT_FORMAT` so that it's clear where the environment variable is consumed (without having to search)